docs/src/basic_usage/querying_data.md

```diff
@@ -0,0 +1,58 @@
+# [Querying Properties](@id querying_data)
+
+Once we have attached the properties to our species using the `@attach_metadata` macro, we can query them using pre-defined functions.
+
+Let us begin by attaching metadata to a species.
+
+```@example ind1
+using PubChem, Catalyst
+
+# Define the species and attach metadata
+@variables t
+@species CH3COOH(t)
```

Comment:
Is there no way to build it as a compound from the PubChem metadata? Can you attach metadata to compounds?

Reply:
(Not such a change should be a separate PR so we can get this one merged.)

Reply:
We can attach metadata to compounds as well. I didn't include the `@compound` macro because the documentation for it is not out yet.